buildspecs/upgrade-eks-distro-buildspec.yml

```diff
@@ -0,0 +1,14 @@
+version: 0.2
```

Comment:
i was wondering why we needed this, but im thinking its because eks-d isnt a traditional project in this repo?

Reply:
Yeah I don't love it, but I needed some way to put it into the upgrade batch buildspec. All the other entries besides the last one are all from the projects directory, so I just used the construct you had for the checksum-pr creation.